pkg/healthz/healthz.go

```diff
@@ -98,7 +98,6 @@ func KubeConnectionHealthzFuncFactory(clusterConfig *virtconfig.ClusterConfig, h
 		res["apiserver"] = map[string]interface{}{"connectivity": "ok", "version": version}
 		res["config-resource-version"] = clusterConfig.GetResourceVersion()
 		response.WriteHeaderAndJson(http.StatusOK, res, restful.MIME_JSON)
-		return
```

Comment:
I wander if there is a specific best practice to avoid adding the `return` statement? If there is, can you ref it?